docs/content/latest/explore/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -126,33 +126,15 @@ Create and populate a table, look at a timestamp to which you'll restore, and th
 
 ### Restore from an absolute time
 
-1. Get a timestamp. YCQL doesn't have a `now()` function, so use a command such as one of the following. You can also use a [YCQL timestamp](../../../api/ycql/type_datetime/#timestamp) with the restore command, if you like.
-
-    ```sh
-    # Ruby: remove the decimal point before using the timestamp
-    $ ruby -e 'puts Time.now.to_f'
-    ```
-
-    ```output
-    1620418801.439626
-    ```
+1. Get a timestamp. YCQL doesn't have a `now()` function, so use a command such as the following. You can also use a [YCQL timestamp](../../../api/ycql/type_datetime/#timestamp) with the restore command, if you like.
```

Comment:
does it make sense to have the YCQL timestamp as an option in the documentation, but not bloat the description here inline?

Reply:
whoops, missed this one. deleted just now. :)